Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -822,18 +826,63 @@ - (void)loadView
 #endif
 
 #if ENABLE(FULLSCREEN_DISMISSAL_GESTURES)
-    _bannerLabel = adoptNS([[_WKInsetLabel alloc] initWithFrame:CGRectMake(0, 0, 100, 100)]);
-    [_bannerLabel setEdgeInsets:UIEdgeInsetsMake(16, 16, 16, 16)];
+    if (_banner) {
+        [_banner removeFromSuperview];
+        _banner = nil;
+        _bannerLabel = nil;
+        _bannerTapToDismissRecognizer = nil;
+    }
```

Comment:
I don't think this logic is necessary. `-loadView` should only be called once during the view controller lifecycle.